model/src/pyrenew/process/rtrandomwalk.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         Rt0_dist: dist.Distribution = dist.TruncatedNormal(
             loc=1.2, scale=0.2, low=0
         ),
-        Rt_transform: AbstractTransform = LogTransform(),
+        Rt_transform: nt.Transform | None = None,
         Rt_rw_dist: dist.Distribution = dist.Normal(0, 0.025),
```

Comment:
Not for this PR, but we should get rid of the defaults here.

Reply:
Created one #145